### ISSUE TITLE
[suggestion] ice4 Options to new Config ?

### ISSUE THREAD

**Issue by DoM1niC** [jitsi/jitsi-videobridge#1662]
Hey I switched my JVB fully on new Config Stuff, so I notice some ice4 Options have to stay in **sip-communicator.properties** like this...
```
org.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS=
org.ice4j.ice.harvest.NAT_HARVESTER_PUBLIC_ADDRESS=
org.ice4j.ice.harvest.DISABLE_AWS_HARVESTER=true
org.ice4j.ice.harvest.STUN_MAPPING_HARVESTER_ADDRESSES=
```
in someday this will be in **conf** stuff too? In my case it's easier to unterstand the fully config and one is better then two :) 

THX 4 this nice Project, I love Jitsi 💯

**Comment by bgrozev** [comment]
Hey @DoM1niC, I completely agree :). 

The good news is that ice4j has been ported to support the new config (see [here](https://github.com/jitsi/ice4j/blob/master/src/main/resources/reference.conf) and [here](https://github.com/jitsi/jitsi-videobridge/blob/master/jvb/src/main/resources/application.conf)), so you can set these in jvb.conf, you just need to figure out the new syntax.

The bad news is that the documentation hasn't been updated.

**Comment by bgrozev** [comment]
Ops, I wrote without confirming -- some of the properties (notably NAT_HARVESTER_) aren't ported yet. But they will be eventually.

**Comment by DoM1niC** [comment]
> Hey @DoM1niC, I completely agree :).
> 
> The good news is that ice4j has been ported to support the new config (see [here](https://github.com/jitsi/ice4j/blob/master/src/main/resources/reference.conf) and [here](https://github.com/jitsi/jitsi-videobridge/blob/master/jvb/src/main/resources/application.conf)), so you can set these in jvb.conf, you just need to figure out the new syntax.
> 
> The bad news is that the documentation hasn't been updated.

Works so far 👍🏻 THX for your fast reply 🥇

**Comment by DoM1niC** [comment]
> Ops, I wrote without confirming -- some of the properties (notably NAT_HARVESTER_) aren't ported yet. But they will be eventually.

Could you tell me here if 
```
org.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS=
org.ice4j.ice.harvest.NAT_HARVESTER_PUBLIC_ADDRESS=
```
 are fully ported and close my issue ?

**Comment by bgrozev** [comment]
Not yet. This PR ports them: jitsi/ice4j#229

**Comment by DoM1niC** [comment]
> Not yet. This PR ports them: [jitsi/ice4j#229](https://github.com/jitsi/ice4j/pull/229)

Looks good :) how this ipv6 works my Coturn is already working with IPv6, so IPv6 needs configure PUBLIC_ADDRESS too ? Or how determine my correct one in JVB ?

**Comment by DoM1niC** [comment]
Fixed / added by #1664
